cloud/scope/common_test.go

```diff
@@ -0,0 +1,183 @@
+package scope
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	mock "github.com/linode/cluster-api-provider-linode/mock"
+	"github.com/linode/linodego"
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/mock/gomock"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
```suggestion
	"github.com/linode/linodego"
	"github.com/stretchr/testify/assert"
	"go.uber.org/mock/gomock"
	corev1 "k8s.io/api/core/v1"
	"k8s.io/apimachinery/pkg/types"
	"sigs.k8s.io/controller-runtime/pkg/client"
	
	"github.com/linode/cluster-api-provider-linode/mock"
```

Reply:
is there a linter that can check for this go import ordering?

Reply:
good question, maybe this might help? https://golangci-lint.run/usage/linters/#gci

Reply:
See #172

Reply:
Fixed!

Reply:
FYI gopls can do this for you on save

https://github.com/golang/tools/tree/master/gopls